CHANGELOG.md

```diff
@@ -1,18 +1,53 @@
 # BuildingSync
 
+## Version 2.1.0
+
+BuildingSync Version 2.1.0 does not introduce any breaking changes; however, a new required element
```

Comment:
...changes; however, the `version` attribute is now required as part of the root `BuildingSync` element....